### PR TITLE
Funnel.io sources part 2

### PR DESCRIPTION
**Pull request by norric-funnel**
(Branched of part 1)

### CODE REVIEW

File: sources/indeed.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "INDEED",
+  "name": "Indeed",
```
Comment (pfrisella) [inline sources/indeed.json]: would it be useful to be more specific here, is it Indeed Ads ?

File: sources/kickback.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "KICKBACK",
+  "name": "KickBack",
+  "categories": ["ADVERTISING"],
+  "organization": "KICKBACK",
+  "iconUrl": "https://static.funnel.io/logos/icon48x48/kickback.png",
+  "sourceUrl": "",
```
Comment (pfrisella) [inline sources/kickback.json]: missing source url

File: organizations/kickback.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "id": "KICKBACK",
+  "name": "KickBack",
+  "orgUrl": "",
```
Comment (pfrisella) [inline organizations/kickback.json]: missing org url

File: organizations/visual_meta.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+  "id": "VISUAL_META",
+  "name": "Visual meta",
```
Comment (pfrisella) [inline organizations/visual_meta.json]: Visual Meta

File: sources/leguide.json [hunk 5]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "LEGUIDE",
+  "name": "LeGuide",
+  "categories": ["ADVERTISING"],
+  "organization": "LEGUIDE",
+  "iconUrl": "https://static.funnel.io/logos/icon48x48/leguide.png",
+  "sourceUrl": "https://www.affili.net/uk/home",
```
Comment (pfrisella) [inline sources/leguide.json]: incorrect sourceUrl ?

File: sources/linkedin.json [hunk 6]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/linkedin.json]: LinkedIn Ads and LinkedIn Analytics sources already exist. Is this a different source of LinkedIn data?

File: organizations/linkedin.json [hunk 7]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/linkedin.json]: not sure this should be an org. can you explain this?

File: sources/metaio.json [hunk 8]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "METAIO",
```
Comment (pfrisella) [inline sources/metaio.json]: META_IO

File: sources/metaio.json [hunk 9]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/metaio.json]: meta_io.json

File: organizations/metaio.json [hunk 10]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/metaio.json]: meta_io.json

File: organizations/metaio.json [hunk 11]
```diff
@@ -0,0 +1,6 @@
...
+  "id": "METAIO",
```
Comment (pfrisella) [inline organizations/metaio.json]: META_IO

File: sources/metaio.json [hunk 12]
```diff
@@ -0,0 +1,9 @@
...
+  "id": "METAIO",
+  "name": "Meta I/O",
+  "categories": ["ADVERTISING"],
+  "organization": "METAIO",
```
Comment (pfrisella) [inline sources/metaio.json]: META_IO

File: sources/channel_pilot.json [hunk 13]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/channel_pilot.json]: looks like name is a single word, so channelpilot.json

File: sources/channel_pilot.json [hunk 14]
```diff
@@ -0,0 +1,9 @@
...
+  "id": "CHANNEL_PILOT",
```
Comment (pfrisella) [inline sources/channel_pilot.json]: CHANNELPILOT

File: sources/channel_pilot.json [hunk 15]
```diff
@@ -0,0 +1,9 @@
...
+  "name": "ChannelPilot",
+  "categories": ["ADVERTISING"],
+  "organization": "CHANNEL_PILOT",
```
Comment (pfrisella) [inline sources/channel_pilot.json]: CHANNELPILOT

File: organizations/channel_pilot.json [hunk 16]
```diff
@@ -0,0 +1,6 @@
+{
+  "id": "CHANNEL_PILOT",
```
Comment (pfrisella) [inline organizations/channel_pilot.json]: CHANNELPILOT

File: organizations/channel_pilot.json [hunk 17]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/channel_pilot.json]: channelpilot.json

File: organizations/cross_install.json [hunk 18]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/cross_install.json]: crossinstall.json

File: organizations/cross_install.json [hunk 19]
```diff
@@ -0,0 +1,6 @@
...
+  "id": "CROSS_INSTALL",
```
Comment (pfrisella) [inline organizations/cross_install.json]: CROSSINSTALL

File: sources/cross_install.json [hunk 20]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/cross_install.json]: crossinstall.json

File: sources/cross_install.json [hunk 21]
```diff
@@ -0,0 +1,9 @@
...
+  "id": "CROSS_INSTALL",
```
Comment (pfrisella) [inline sources/cross_install.json]: CROSSINSTALL

File: sources/cross_install.json [hunk 22]
```diff
@@ -0,0 +1,9 @@
...
+  "name": "CrossInstall",
+  "categories": ["ADVERTISING"],
+  "organization": "CROSS_INSTALL",
```
Comment (pfrisella) [inline sources/cross_install.json]: CROSSINSTALL

File: sources/polyvore.json [hunk 23]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "POLYVORE",
+  "name": "Polyvore",
+  "categories": [""],
```
Comment (pfrisella) [inline sources/polyvore.json]: missing categories

File: sources/polyvore.json [hunk 24]
```diff
@@ -0,0 +1,9 @@
...
+  "organization": "POLYVORE",
+  "iconUrl": "https://static.funnel.io/logos/icon48x48/polyvore.png",
+  "sourceUrl": "",
```
Comment (pfrisella) [inline sources/polyvore.json]: missing sourceUrl

File: organizations/polyvore.json [hunk 25]
```diff
@@ -0,0 +1,6 @@
+{
+  "id": "POLYVORE",
+  "name": "Polyvore",
+  "orgUrl": "",
```
Comment (pfrisella) [inline organizations/polyvore.json]: missing orgUrl

File: sources/postaffiliate.json [hunk 26]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/postaffiliate.json]: post_affiliate.json

File: organizations/postaffiliate.json [hunk 27]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/postaffiliate.json]: post_affiliate.json

File: organizations/postaffiliate.json [hunk 28]
```diff
@@ -0,0 +1,6 @@
...
+  "id": "POSTAFFILIATE",
```
Comment (pfrisella) [inline organizations/postaffiliate.json]: POST_AFFILIATE

File: sources/postaffiliate.json [hunk 29]
```diff
@@ -0,0 +1,9 @@
...
+  "id": "POSTAFFILIATE",
+  "name": "Post Affiliate",
+  "categories": ["ADVERTISING"],
+  "organization": "POSTAFFILIATE",
```
Comment (pfrisella) [inline sources/postaffiliate.json]: POST_AFFILIATE

File: sources/postaffiliate.json [hunk 30]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "POSTAFFILIATE",
```
Comment (pfrisella) [inline sources/postaffiliate.json]: POST_AFFILIATE

File: sources/prisguide.json [hunk 31]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "PRISGUIDE",
+  "name": "Prisguide",
```
Comment (pfrisella) [inline sources/prisguide.json]: should this be Prisguiden ?

File: organizations/prisguide.json [hunk 32]
```diff
@@ -0,0 +1,6 @@
+{
+  "id": "PRISGUIDE",
+  "name": "Prisguide",
```
Comment (pfrisella) [inline organizations/prisguide.json]: should this be Prisguiden ?

File: sources/rakuten.json [hunk 33]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "RAKUTEN",
+  "name": "Rakuten LinkShare",
```
Comment (pfrisella) [inline sources/rakuten.json]: not sure but is it worth calling this RAKUTEN_MARKETING or RAKUTEN_LINKSHARE ?

File: sources/reddit_ads.json [hunk 34]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "REDDIT_ADS",
+  "name": "Reddit",
```
Comment (pfrisella) [inline sources/reddit_ads.json]: Reddit Ads

File: sources/rocketfuel.json [hunk 35]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/rocketfuel.json]: rocket_fuel.json

File: sources/rocketfuel.json [hunk 36]
```diff
@@ -0,0 +1,9 @@
...
+  "id": "ROCKETFUEL",
```
Comment (pfrisella) [inline sources/rocketfuel.json]: ROCKET_FUEL

File: organizations/sizmekdsp.json [hunk 37]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/sizmekdsp.json]: Seems like you should only have 1 SIZMEK org ?

File: sources/rtbhouse.json [hunk 38]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/rtbhouse.json]: rtb_house.json

File: organizations/rtbhouse.json [hunk 39]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/rtbhouse.json]: rtb_house.json

File: sources/rtbhouse.json [hunk 40]
```diff
@@ -0,0 +1,9 @@
...
+  "id": "RTBHOUSE",
```
Comment (pfrisella) [inline sources/rtbhouse.json]: RTB_HOUSE

File: sources/rtbhouse.json [hunk 41]
```diff
@@ -0,0 +1,9 @@
...
+  "name": "RTB House",
+  "categories": ["ADVERTISING"],
+  "organization": "RTBHOUSE",
```
Comment (pfrisella) [inline sources/rtbhouse.json]: RTB_HOUSE

File: organizations/rtbhouse.json [hunk 42]
```diff
@@ -0,0 +1,6 @@
...
+  "id": "RTBHOUSE",
```
Comment (pfrisella) [inline organizations/rtbhouse.json]: RTB_HOUSE

File: sources/sizmek.json [hunk 43]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "SIZMEK",
```
Comment (pfrisella) [inline sources/sizmek.json]: should this be SIZMEK_MDX ?

File: sources/sizmekdsp.json [hunk 44]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "SIZMEKDSP",
```
Comment (pfrisella) [inline sources/sizmekdsp.json]: SIZMEK_DSP

File: sources/sizmekdsp.json [hunk 45]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/sizmekdsp.json]: sizmek_dsp.json

File: sources/sizmekdsp.json [hunk 46]
```diff
@@ -0,0 +1,9 @@
...
+  "id": "SIZMEKDSP",
+  "name": "Sizmek DSP",
+  "categories": ["ADVERTISING"],
+  "organization": "SIZMEKDSP",
```
Comment (pfrisella) [inline sources/sizmekdsp.json]: Should this be SIZMEK ?

File: organizations/sizmek.json [hunk 47]
```diff
@@ -0,0 +1,6 @@
+{
+  "id": "SIZMEK",
+  "name": "Sizmek MDX",
```
Comment (pfrisella) [inline organizations/sizmek.json]: Just Sizmek ? seems like there should be just 1 SIZMEK org

File: organizations/snapchat.json [hunk 48]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/snapchat.json]: should this be just SNAP and Snap Inc ?

File: sources/snapchat.json [hunk 49]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "SNAPCHAT",
+  "name": "Snapchat",
+  "categories": ["ADVERTISING"],
+  "organization": "SNAPCHAT",
```
Comment (pfrisella) [inline sources/snapchat.json]: SNAP, assuming org is also updated.

File: sources/snapchat.json [hunk 50]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "SNAPCHAT",
+  "name": "Snapchat",
```
Comment (pfrisella) [inline sources/snapchat.json]: is there a more specific name for data source, is this Snapchat Ads?

File: sources/spotashop.json [hunk 51]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/spotashop.json]: spot_a_shop.json

File: organizations/spotashop.json [hunk 52]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/spotashop.json]: spot_a_shop.json

File: sources/spotashop.json [hunk 53]
```diff
@@ -0,0 +1,9 @@
...
+  "id": "SPOTASHOP",
```
Comment (pfrisella) [inline sources/spotashop.json]: SPOT_A_SHOP

File: sources/spotashop.json [hunk 54]
```diff
@@ -0,0 +1,9 @@
...
+  "name": "Spot-A-Shop",
+  "categories": ["ECOMMERCE"],
+  "organization": "SPOTASHOP",
```
Comment (pfrisella) [inline sources/spotashop.json]: SPOT_A_SHOP

File: organizations/spotashop.json [hunk 55]
```diff
@@ -0,0 +1,6 @@
...
+  "id": "SPOTASHOP",
```
Comment (pfrisella) [inline organizations/spotashop.json]: SPOT_A_SHOP

File: sources/trace_delight.json [hunk 56]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/trace_delight.json]: tracdelight.json

File: organizations/trace_delight.json [hunk 57]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/trace_delight.json]: tracdelight.json

File: sources/trace_delight.json [hunk 58]
```diff
@@ -0,0 +1,9 @@
...
+  "id": "TRACE_DELIGHT",
```
Comment (pfrisella) [inline sources/trace_delight.json]: TRACDELIGHT

File: organizations/trace_delight.json [hunk 59]
```diff
@@ -0,0 +1,6 @@
...
+  "id": "TRACE_DELIGHT",
```
Comment (pfrisella) [inline organizations/trace_delight.json]: TRACDELIGHT

File: sources/unityads.json [hunk 60]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/unityads.json]: unity_ads.json

File: organizations/unityads.json [hunk 61]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/unityads.json]: the org should just be Unity or Unity Technologies. so unity.json

File: organizations/unityads.json [hunk 62]
```diff
@@ -0,0 +1,6 @@
...
+  "id": "UNITYADS",
```
Comment (pfrisella) [inline organizations/unityads.json]: UNITY

File: organizations/unityads.json [hunk 63]
```diff
@@ -0,0 +1,6 @@
...
+  "name": "Unity Ads",
```
Comment (pfrisella) [inline organizations/unityads.json]: Unity

File: sources/unityads.json [hunk 64]
```diff
@@ -0,0 +1,9 @@
...
+  "id": "UNITYADS",
```
Comment (pfrisella) [inline sources/unityads.json]: UNITY_ADS

File: sources/unityads.json [hunk 65]
```diff
@@ -0,0 +1,9 @@
...
+  "name": "Unity Ads",
+  "categories": ["ADVERTISING"],
+  "organization": "UNITYADS",
```
Comment (pfrisella) [inline sources/unityads.json]: UNITY

File: organizations/unityads.json [hunk 66]
```diff
@@ -0,0 +1,6 @@
...
+  "orgUrl": "https://unity3d.com/unity/features/ads",
```
Comment (pfrisella) [inline organizations/unityads.json]: https://unity3d.com

File: sources/veinteractive.json [hunk 67]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/veinteractive.json]: ve_interactive.json

File: organizations/veinteractive.json [hunk 68]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/veinteractive.json]: ve_interactive.json

File: organizations/veinteractive.json [hunk 69]
```diff
@@ -0,0 +1,6 @@
...
+  "id": "VEINTERACTIVE",
```
Comment (pfrisella) [inline organizations/veinteractive.json]: VE_INTERACTIVE

File: sources/veinteractive.json [hunk 70]
```diff
@@ -0,0 +1,9 @@
...
+  "id": "VEINTERACTIVE",
```
Comment (pfrisella) [inline sources/veinteractive.json]: VE_INTERACTIVE

File: sources/veinteractive.json [hunk 71]
```diff
@@ -0,0 +1,9 @@
...
+  "name": "Ve Interactive",
+  "categories": ["ADVERTISING"],
+  "organization": "VEINTERACTIVE",
```
Comment (pfrisella) [inline sources/veinteractive.json]: VE_INTERACTIVE

File: sources/vkads.json [hunk 72]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/vkads.json]: vk_ads.json

File: sources/vkads.json [hunk 73]
```diff
@@ -0,0 +1,9 @@
...
+  "id": "VKADS",
```
Comment (pfrisella) [inline sources/vkads.json]: VK_ADS

File: sources/vkads.json [hunk 74]
```diff
@@ -0,0 +1,9 @@
...
+  "name": "VK Ads",
+  "categories": ["SOCIAL"],
+  "organization": "VKADS",
```
Comment (pfrisella) [inline sources/vkads.json]: VK

File: organizations/vkads.json [hunk 75]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/vkads.json]: the org should just be vk. So vk.json

File: organizations/vkads.json [hunk 76]
```diff
@@ -0,0 +1,6 @@
...
+  "id": "VKADS",
```
Comment (pfrisella) [inline organizations/vkads.json]: VK

File: organizations/vkads.json [hunk 77]
```diff
@@ -0,0 +1,6 @@
...
+  "name": "VK Ads",
```
Comment (pfrisella) [inline organizations/vkads.json]: VK

File: organizations/yandex_display.json [hunk 78]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/yandex_display.json]: there is already a yandex org
https://github.com/googledatastudio/ds-data-registry/blob/master/organizations/yandex.json

File: sources/yandex_display.json [hunk 79]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "YANDEX_DISPLAY",
+  "name": "Yandex Display",
+  "categories": ["ADVERTISING"],
+  "organization": "YANDEX_DISPLAY",
```
Comment (pfrisella) [inline sources/yandex_display.json]: this should be YANDEX
and the org is alread defined so you don't need to add it.
https://github.com/googledatastudio/ds-data-registry/blob/master/organizations/yandex.json

File: sources/yelp.json [hunk 80]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "YELP",
```
Comment (pfrisella) [inline sources/yelp.json]: will defer to you but confirming that this is not a more specific data source, E.g. Yelp Ads ?
https://biz.yelp.com/advertise

File: sources/indeed.json (hunk 1 above)
Comment (norric-funnel) [reply sources/indeed.json]: It's job ads, which appears to be all they do. I've precised the category instead

**Pull request merged**